project/Dependencies.scala

```diff
@@ -7,7 +7,7 @@ object Dependencies {
   val awsClientV2Version = "2.16.86"
 
   val sentryLogback = "io.sentry" % "sentry-logback" % "1.7.5"
-  val identityAuth = "com.gu.identity" %% "identity-auth-play" % "3.255"
+  val identityAuth = "com.gu.identity" %% "identity-auth-play" % "3.256.03-SNAPSHOT"
```

Comment:
this is just using the snapshot version of the library for testing in `CODE` (the `.03` at the end of the library version is because I released 3 additional snapshot versions to test but that should not affect the version number of the library so I didn't push the version change in the identity PR as the released version should just be `3.256`)